### PR TITLE
Artist Profile Updates

### PR DESCRIPTION
**Pull request by emilyzhong**
## Artist Profile Updates
* Change "Edit profile" button to an "edit" button (persisted, not hovered)
* Using loading component on load
* Add a prompt to finish profile
https://app.asana.com/0/860297512787618/1117511071564526/f

### Related PRs
None

### Migrations
None

### Tests Performed, Edge Cases
None

### Screenshots
image

CC: @by-co

### CODE REVIEW

**Comment by by-co** [comment]
Hmm for some reason it's not letting me click "Edit"
Screen Recording 2019-04-10 at 03 53 PM

**Comment by emilyzhong** [comment]
should be fixed now — both the issues were with the dropdown being "present" + covering the edit button even when not hovered over

File: app/javascript/components/artists/ArtistProfile.jsx [hunk 1]
```diff
@@ -122,6 +124,65 @@ class ArtistProfile extends React.Component {
...
+        completionStatus[x] ? -1 : 1);
+    });
+
+    let numCompleted = 0;
+    for (let i = 0; i < sortedKeys.length; i++) {
+      if (completionStatus[sortedKeys[i]]) { numCompleted++ }
+    }
+
+    if (numCompleted == sortedKeys.length) {
+      this.setState({showIncompleteBanner: false})
```
Comment (by-co) [inline app/javascript/components/artists/ArtistProfile.jsx]: Hmm if you open up console it throws an error about this line
```
Warning: Cannot update during an existing state transition (such as within `render`). Render methods should be a pure function of props and state.
```

**Comment by by-co** [comment]
Do you also think you could debug the overlay issue here? It's present when logging in as admin, also

Screen Recording 2019-04-12 at 04 42 PM

**Comment by by-co** [comment]
ALSO last thing, the "sold" overlay on `WorkPanelColumn` seems to be blocking users from clicking on artwork thumbnails to navigate to details :( Do you think you could fix? This is the last PR going into release 1.2 so thanks for bearing w all the random buggos T_T
